src/unitxt/fusion.py

```diff
@@ -12,19 +11,21 @@ class BaseFusion(SourceOperator):
     """BaseFusion operator that combines multiple streams into one.
 
     Args:
-        include_splits: List of splits to include. If None, all splits are included.
+        origins: a dict of named SourceOperators
+        include_splits: List of splits to include from each SourceOperator.
+                If None, all splits are included.
     """
 
-    origins: List[SourceOperator]
+    origins: Dict[str, SourceOperator]
```

Comment:
```suggestion
    origins: Union[List[SourceOperato], Dict[str, SourceOperator]]
```
I think we should allow also anonymus sources merging which are merged without given name per source

Reply:
And in such a case, nothing will be written into field "group" ?  or I will generate juicy names like "0", "1", "2" , and store them into field "group"?